### PR TITLE
Several fixes

### PR DESCRIPTION
**Pull request by boscohyun**
- Remove useless files.
- Improve null check and add log.
- Add missed trigger in CharacterAnimator.

### CODE REVIEW

File: .github/workflows/pack.yml [hunk 1]
```diff
@@ -34,10 +34,10 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.8.6
-      - uses: actions/cache@v2
-        with:
-          path: ${{ matrix.projectPath }}/Library
-          key: ${{ matrix.targetPlatform }}-${{ matrix.projectPath }}-Library-v2
+      # - uses: actions/cache@v2
```
Comment (hskim881028) [inline .github/workflows/pack.yml]: Are we not using the cache function anymore?

Comment (boscohyun) [reply .github/workflows/pack.yml]: Yes we use. This differ does not remained anymore.

**Review by Unengine — APPROVED** [review]

**Review by hskim881028 — APPROVED** [review]

**Comment by hskim881028** [comment]
ci is failed

**Comment by boscohyun** [comment]
> ci is failed

We don't use circle ci anymore so.. ignore this time.😅

**Pull request merged**